backend/src/middleware/slugify/uniqueSlug.js

```diff
@@ -1,6 +1,6 @@
 import slugify from 'slug'
 export default async function uniqueSlug (string, isUnique) {
-  let slug = slugify(string, {
+  let slug = slugify(string || 'anonymous', {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:39:43Z" title="Thursday, May 2nd 2019, 7:39:43 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,6 @@
 import slugify from 'slug'
 export default async function uniqueSlug (string, isUnique) {
-  let slug = slugify(string, {
+  let slug = slugify(string || 'anonymous', {
```


```suggestion
  let slug = slugify(string || 'Anonymous', {
```

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T10:56:03Z" title="Monday, May 6th 2019, 12:56:03 pm +02:00">May 6, 2019</time>_

---


Greg said slugs are lower case